tests/acceptance/SubscriberManageImportExportCest.php

```diff
@@ -9,21 +9,18 @@ function importUsersToSubscribersViaCSV(\AcceptanceTester $I) {
     $I->wantTo('Import a subscriber list from CSV');
     $I->login();
     $I->amOnMailPoetPage ('Subscribers');
-    //click import
     $I->click('[data-automation-id="import-subscribers-button"]');
-    $I->waitForText('Back to Subscribers');
-    //select upload file as import method, import CSV
-    $I->click('[data-automation-id="import-csv-method"]');
-    $I->attachFile('[data-automation-id="import-file-upload-input"]', 'MailPoetImportList.csv');
-    $I->click('#method_file [data-automation-id="import-next-step"]');
+    $this->uploadCsvFile($I);
+    $this->chooseListAndConfirm($I);
+    $I->wait(10);
```

Comment:
Do we really have to have this long explicit wait here? And twice in this test? This will make every single test run longer for 20 seconds. Couldn't we wait for the next text to appear?

Reply:
Aaah, I forgot this, it was for debugging. I'll remove it.